client/src/pages/CreateJob.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+
+const CreateJob = () => {
+  const [jobTitle, setJobTitle] = useState<string>('');
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setJobTitle(event.target.value)
+  }
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    console.log('Job Title:', jobTitle )
+  }
+
+  return (
+    <div>
+      <form action="">
+        <input 
+          type="text"
+          value={jobTitle}
+          onChange={handleInputChange} 
+          placeholder="Enter job title" />
+        <button type="submit">Create Job</button>
+      </form>
+      
+      <p>Job Title: {jobTitle}</p>
```

Comment:
A task should be defined by multiple things, not just the title.
- Title
- Description
- Location (can be just text)
- Price
- By when should it be done.
- Images and media that the user would upload.

Some of the fields can be optinal.

Reply:
I think this one is done as well. All the attributes are in the form.